kinds/structured/team/team_kind.cue

```diff
@@ -0,0 +1,26 @@
+package kind
+
+name: "Team"
+maturity: "merged"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// v0.0
+			// TODO docs
+			{
+				id?: int
+				orgId?: int @grafanamaturity(MaybeRemove)
+				name: string
+				email: string
+				avatarUrl: string @grafanamaturity(MaybeRemove)
+				memberCount: int @grafanamaturity(MaybeRemove)
+				permission: #Permission @grafanamaturity(MaybeRemove)
+				accessControl: [string]: bool @grafanamaturity(MaybeRemove)
```

Comment:
Most of these fields are annotated with `grafanamaturity(MaybeRemove)` attribute cause I'm having a hard time figuring out how the schema should look like given:
- It looks like only two fields are sufficient to create a new team:
```
type CreateTeamCommand struct {
	Name  string `json:"name" binding:"Required"`
	Email string `json:"email"`
	OrgId int64  `json:"-"`
	Result Team `json:"-"`
}
```

- There are a few `Team` types in the codebase with slightly different list of fields
```
// Team model - this one matches the team table schema in the database
type Team struct {
	Id    int64  `json:"id"`
	OrgId int64  `json:"orgId"`
	Name  string `json:"name"`
	Email string `json:"email"`

	Created time.Time `json:"created"`
	Updated time.Time `json:"updated"`
}

type TeamDTO struct {
	Id            int64           `json:"id"`
	OrgId         int64           `json:"orgId"`
	Name          string          `json:"name"`
	Email         string          `json:"email"`
	AvatarUrl     string          `json:"avatarUrl"`
	MemberCount   int64           `json:"memberCount"`
	Permission    PermissionType  `json:"permission"`
	AccessControl map[string]bool `json:"accessControl"`
}
```

Reply:
I also wonder why this field generates TS type as `Record<string, unknown>`, while it'd be `Record<string, boolean>`?

Reply:
> I also wonder why this field generates TS type as `Record<string, unknown>`, while it'd be `Record<string, boolean>`?

Hmm, that's interesting. Might be a bug in cuetsy.

Reply:
> > I also wonder why this field generates TS type as `Record<string, unknown>`, while it'd be `Record<string, boolean>`?
> 
> Hmm, that's interesting. Might be a bug in cuetsy.

It also generates the Go type as `map[string]interface{}` but probably should be `map[string]bool`.